operators/percona-xtradb-cluster-operator/1.15.0/manifests/percona-xtradb-cluster-operator.v1.15.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,742 @@
+                - perconaxtradbclusters
+                - perconaxtradbclusters/status
+                - perconaxtradbclusterbackups
+                - perconaxtradbclusterbackups/status
+                - perconaxtradbclusterrestores
+                - perconaxtradbclusterrestores/status
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - pods/exec
+                - pods/log
+                - configmaps
+                - services
+                - persistentvolumeclaims
+                - secrets
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+                - replicasets
+                - statefulsets
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - batch
+              resources:
+                - jobs
+                - cronjobs
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - policy
+              resources:
+                - poddisruptionbudgets
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - coordination.k8s.io
+              resources:
+                - leases
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - events
+              verbs:
+                - create
+                - patch
+            - apiGroups:
+                - certmanager.k8s.io
+                - cert-manager.io
+              resources:
+                - issuers
+                - certificates
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - update
+                - patch
+                - delete
+                - deletecollection
+          serviceAccountName: percona-xtradb-cluster-operator
+        deployments: []
+  installModes:
```

Comment:
@jvpasinatto could you please update installModes like this: 
`  installModes:
    - {type: OwnNamespace, supported: true}
    - {type: SingleNamespace, supported: true}
    - {type: MultiNamespace, supported: false}
    - {type: AllNamespaces, supported: true}`